README.md

```diff
@@ -3,44 +3,46 @@
+| Spender                          | [0x3c68dfc45dc92C9c605d92B49858073e10b857A6](https://etherscan.io/address/0x3c68dfc45dc92C9c605d92B49858073e10b857A6) | Tokenlon         |
+| AllowanceTarget                  | [0x8A42d311D282Bfcaa5133b2DE0a8bCDBECea3073](https://etherscan.io/address/0x8A42d311D282Bfcaa5133b2DE0a8bCDBECea3073) | Tokenlon         |
+| PMM                              | [0x8D90113A1e286a5aB3e496fbD1853F265e5913c6](https://etherscan.io/address/0x8D90113A1e286a5aB3e496fbD1853F265e5913c6) | Tokenlon         |
+| AMMQuoter                        | [0x7839254CfF8aaFBdC2da66fe709eB8f17cE09fe5](https://etherscan.io/address/0x7839254CfF8aaFBdC2da66fe709eB8f17cE09fe5) | Tokenlon         |
+| AMMWrapperWithPath               | [0x4a14347083B80E5216cA31350a2D21702aC3650d](https://etherscan.io/address/0x4a14347083B80E5216cA31350a2D21702aC3650d) | Tokenlon         |
+| RFQ                              | [0xfD6C2d2499b1331101726A8AC68CCc9Da3fAB54F](https://etherscan.io/address/0xfD6C2d2499b1331101726A8AC68CCc9Da3fAB54F) | Tokenlon         |
+| xLON                             | [0xf88506b0f1d30056b9e5580668d5875b9cd30f23](https://etherscan.io/address/0xf88506b0f1d30056b9e5580668d5875b9cd30f23) | Staking          |
+| LONStaking (Logic contract)      | [0x413ecce5d56204962090eef1dead4c0a247e289b](https://etherscan.io/address/0x413ecce5d56204962090eef1dead4c0a247e289b) | Staking          |
+| MiningTreasury                   | [0x292a6921Efc261070a0d5C96911c102cBF1045E4](https://etherscan.io/address/0x292a6921Efc261070a0d5C96911c102cBF1045E4) | Mining Reward    |
+| TreasuryVesterFactory            | [0x000000003A8DBF47cD362EDA39B3a5F3FC6E99ce](https://etherscan.io/address/0x000000003A8DBF47cD362EDA39B3a5F3FC6E99ce) | Vesting          |
+| MerkleRedeem                     | [0x0000000006a0403952389B70d8EE4E45479023db](https://etherscan.io/address/0x0000000006a0403952389B70d8EE4E45479023db) | Reward           |
+| RewardDistributor                | [0xbF1C2c17CC77e7Dec3466B96F46f93c09f02aB07](https://etherscan.io/address/0xbF1C2c17CC77e7Dec3466B96F46f93c09f02aB07) | Buyback          |
+| StakingRewards (LON/ETH)         | [0xb6bC1a713e4B11fa31480d31C825dCFd7e8FaBFD](https://etherscan.io/address/0xb6bC1a713e4B11fa31480d31C825dCFd7e8FaBFD) | Liquidity mining |
+| StakingRewards (LON/USDT)        | [0x9648B119f442a3a096C0d5A1F8A0215B46dbb547](https://etherscan.io/address/0x9648B119f442a3a096C0d5A1F8A0215B46dbb547) | Liquidity mining |
 
 ## Prerequisite
 
 -   node (>=14.0.0 <16)
 -   yarn (^1.22.10)
 -   [foundry](https://github.com/foundry-rs/foundry)
 -   Environment Variables (Used for foundry fork tests)
-    -   MAINNET_NODE_RPC_URL : The RPC URL for accessing forked states.
-    -   FORK_BLOCK_NUMBER : Specfic block number of forked states.
+    -   `MAINNET_NODE_RPC_URL`: The RPC URL for accessing forked states.
+    -   `FORK_BLOCK_NUMBER`: Specfic block number of forked states.
```

Comment:
@changwu-tw Maybe we can merge the examples in `.env.example` to here?

Reply:
Yes, fixed it in [219a498](https://github.com/consenlabs/tokenlon-contracts/pull/22/commits/219a498463fedc10513fe39adc5a279457fdb38d)